packages/plugin-ext/src/common/proxy-handler.ts

```diff
@@ -69,13 +78,13 @@ export class ClientProxyHandler<T extends object> implements ProxyHandler<T> {
         const isNotify = this.isNotification(name);
         return (...args: any[]) => {
             const method = name.toString();
-            return this.rpcDeferred.promise.then(async (connection: RpcProtocol) => {
+            return this.proxySynchronizer.pendingProxyInitializations().then(() => this.rpcDeferred.promise.then(async (connection: RpcProtocol) => {
```

Comment:
Should we make this into an async function to make the code look a little bit nicer? Doesn't have to be done, just a thought.